sos/plugins/dnf.py

```diff
@@ -60,4 +66,17 @@ def setup(self):
         if self.get_option("history"):
             self.add_cmd_output("dnf history")
 
+        if self.get_option("history-info"):
+            history = self.call_ext_prog("dnf history")
```

Comment:
Isn't that what lines 66-67 already do?

If you mean "do that even if user didn't pass the option 'history'", then I might agree, but:
1. Will let sos maintainer decide, not sure what's the best way to do that. I don't mind pushing an updated patch if needed.
2. It seems to already be how it works, see my previous comment - '-k dnf.history-info=on' didn't work for me, only '-k dnf.history-info=on -k dnf.history=on' did. Not sure that's intentional, or a by-product of how options are handled.